### DISCUSSION TITLE
Visualize and add contours in DICOM

### DISCUSSION THREAD

**Discussion by Mohammad-debug** [Project-MONAI/MONAILabel#1153] [General]
Hi all,
A) I am looking for a tool similar to [cvat ](https://github.com/opencv/cvat) but for DICOM images, Do we have something similar to cvat for DICOM in MONAI?

B) Secondly, Do we have a tool in MONAI to visualize and add contouring in DICOM images?

**Comment by tangy5** [comment]
Hi @Mohammad-debug , MONAI Label supports OHIF, a web-based viewer and can use Orthanc as Dicom-web server. 

more details: https://docs.monai.io/projects/label/en/latest/quickstart.html#setup-development-dicom-server

**Comment by diazandr3s** [comment]
> Hi all, A) I am looking for a tool similar to [cvat ](https://github.com/opencv/cvat) but for DICOM images, Do we have something similar to cvat for DICOM in MONAI?
> 
> B) Secondly, Do we have a tool in MONAI to visualize and add contouring in DICOM images?

As @tangy5 mentioned, OHIF could be an option for this. 
You should also be able to use 3D Slicer with MONAI Label to annotate DICOM images. You'll need to have a DICOM Web server (i.e. Orthanc) to store the DICOM files.
The MONAI Label modules are not tailored to work with contours, but there is no limitation to adding this feature to the reference apps.

Hope this helps,
